cli/config/config.go

```diff
@@ -43,6 +43,7 @@ type ImageDefinition struct {
 	Base    string                 `yaml:"base"`
 	Version string                 `yaml:"version"`
 	Exec    string                 `yaml:"exec"`
+	PreExec string                 `yaml:"preExec"`
```

Comment:
Hm. I guess we have a choice between this being a long string or an array (doing it like gitlab):
```
preExec:
  - echo hello
  - echo world
```

Thought about any preferences or caveats here?

Reply:
No, I actually wanted to discuss this